src/main/java/edu/umich/its/cpm/Utils.java

```diff
@@ -624,7 +618,8 @@ public static String sanitizeName(String type, String name) {
 	}
     // Windows Explorer don't like below special characters in Folder names so zip extraction fails. Replacing with _
 	public static String sanitizeFolderNames(String zipFileName) {
-		return zipFileName.replaceAll("[?>|<:]", "_");
+		zipFileName=zipFileName.replace('\\','_');
```

Comment:
Clarify if the double slash in replace deals with a double slash in the actual string or if the double slash is required to escape the escape character.  It will save time for people reading the code later.

Do we know that there won't be multiple occurrences of '\\' in the name?